.drone.yml

```diff
@@ -13,16 +13,14 @@ pipeline:
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/1/ && cd /tmp/1/
-      - ./project/scripts/sbt testAll
-      - ./project/scripts/sbt ";dotty-bench/jmh:run 1 1 tests/run/arrays.scala"
+      - ./project/scripts/sbt compile testAll "dotty-bench/jmh:run 1 1 tests/pos/alias.scala"
```

Comment:
This does not work unfortunately. The `project/script/sbt` script expects one argument. Either use the old syntax or change the script

Reply:
Also, do we have to run the `dotty-bench` project? Can we just compile it?

Reply:
No, I remembered that I added it because at some point we had a runtime bug on that project that stayed undetected for a while.